### PR TITLE
Revise RDF data models

### PR DESCRIPTION
**Pull request by arnikz**
Updated input data and RDF models (TYPE annotations only) as documented [here](https://github.com/LINNAE-project/SFB-Annotator/tree/iss81/doc).

### CODE REVIEW

**Comment by lisestork** [comment]
The visualisations look really great. Below some comments:

**1_1:** 
- Only evidence (in the form of a token, which is a) an image of an observation, and b) subimage of a measurement or fact such as a table or an anatomical entity) can be used with 'derivedFrom'. So dsw:derivedFrom should be removed here.
- Identification should not be a taxon, because then it would be a taxon class, the identification refers to the process of identification, so should receive a minted URI. See example 22 in https://dwc.tdwg.org/rdf/. In my opinion, we could mint a for the identification (or have a blank node for now) and link the triple gbif:9180402 dwc:taxonRank obo:TAXRANK_0000006 to the body of the annotation with the link owl:sameAs, indicating that we believe this taxon that is annotated is in fact the same as the one that is described by gbif. I believe it would be very important to trace the provenance of such statements, possibly using nanopublications. But that would be a next step  
- I am doubtful whether something can be both a class (e.g. dwc:Taxon) and a textualbody, although the graph seems to make a lot of sense like this. 

**2_1:** 
- same as last point 1_1 (semantic class vs textual body)

**3_1:**
- same as last point 1_1 (semantic class vs textual body)

**4_1:**
- same as last point 1_1 (semantic class vs textual body)
- Here, dsw:derivedFrom is used well, as dwc:Measurement is seen as a token. So, additionally, this node could have dsw:Token as its class. 

**5_1:** 
- Here, what we annotate should be (in my opinion) of type measurementType. So, it would not be true that the body is a measurement. However, I think that you are right in modelling it like this, we should possibly prefer pragmatism over being ontologically correct. Also, no class exists for MeasurementType. If we would like, however, we could create a class MeasurementType, and say: 
bodyIRI -> rdf:type -> MeasurementType. 
measurementOrfactIRI -> dwc:measurementType -> body1. 
measurementOrfactIRI -> rdf:type -> dwc:MeasurementOrFact. 
Let's discuss this further. 

**6_1:** 
- same as 5_1

**7_1:** 
- I think here we should say that the body is a date, and link the conceptual event to the date with either verbatimEvent or dwc:eventDate

File: data/rdf/local/example_1_1.ttl [hunk 1]
```diff
@@ -2,48 +2,52 @@
...
 
 orcid:0000-0002-2146-4803 a foaf:Person .
 
 img:MMNAT01_AF_NNM001001033_003.jpg a dcmitype:StillImage, dsw:Token, dwc:HumanObservation,
     oa:Source, foaf:Image .
 
-taxon:9397 a dwc:Taxon;
-  dwc:taxonRank obo:TAXRANK_0000003 .
+gbif:9180402 dwc:taxonRank obo:TAXRANK_0000006;
+  owl:sameAs _:node1evgqka3hx9 .
```
Comment (lisestork) [inline data/rdf/local/example_1_1.ttl]: As `gbif:9180402` `owl:sameAs` `_:node1evgqka3hx9` is a reification and not a fact, could we add this as a reification statement? Would be an interesting experiment. The modelling seems a bit more straightforward in RDF*, but still this seems like a good way to keep track of provenance of statements. 

ex:statementIRI a rdf:Statement;
      dcterms:creator orcid:0000-0002-2146-4803;
      rdf:subject gbif:9180402;
      rdf:predicate owl:sameAs;
      rdf:object _:node1evgqka3hx9 .

**Review by lisestork — COMMENTED** [review]
Everything looks very good. I just had one comment which we could discuss, otherwise I will approve the pull request.

File: data/rdf/local/example_1_1.ttl (hunk 1 above)
Comment (arnikz) [reply data/rdf/local/example_1_1.ttl]: See #85.

**Comment by arnikz** [comment]
@lisestork: I've updated the [example 1_1](https://github.com/LINNAE-project/SFB-Annotator/blob/iss81/doc/models/example_1_1.svg) as discussed, however, the following triples are currently no so useful IMO:smile:

https://github.com/LINNAE-project/SFB-Annotator/blob/5d2a07442d5b0d8f2cdb21a16d0087e16efe8ebc/data/rdf/local/example_1_1.ttl#L51-L52

Perhaps these triples could be added later when connecting two or more TYPE annotations. What do you think?

**Comment by lisestork** [comment]
> @lisestork: I've updated the [example 1_1](https://github.com/LINNAE-project/SFB-Annotator/blob/iss81/doc/models/example_1_1.svg) as discussed, however, the following triples are currently no so useful IMO😄
> 
> https://github.com/LINNAE-project/SFB-Annotator/blob/5d2a07442d5b0d8f2cdb21a16d0087e16efe8ebc/data/rdf/local/example_1_1.ttl#L51-L52
> 
> Perhaps these triples could be added later when connecting two or more TYPE annotations. What do you think?

I agree, they could be added later when connecting TYPE annotations.

**Review by lisestork — APPROVED** [review]

**Pull request merged**